config/Version.public.xcconfig

```diff
@@ -1,4 +1,4 @@
 VERSION_SHORT=1.6
 
 // Public long version example: VERSION_LONG=9.9.0.0
-VERSION_LONG=1.6.0.1
+VERSION_LONG=1501
```

Comment:
@loremattei I have to change the version format here to avoid AppStore Connect rejecting the version saying that 1.6.0.1 is smaller than the existing version (1500). We will need to figure out how to update the tooling for this.

Reply:
@jtreanor sounds good! I think the version here should be `1601` though as the "old" style naming was to use the long version without the dots.

Reply:
Yeah, I realised that after I submitted. I think we can leave it this time 😄

Reply:
Yep! If you already submitted, I agree that we can live with it :-)